librz/include/rz_search.h

```diff
@@ -213,14 +255,22 @@ typedef enum {
 	RZ_SEARCH_COLLECTION_CRYPTOGRAPHIC_ECC,
 	RZ_SEARCH_COLLECTION_CRYPTOGRAPHIC_SAFECURVES,
 	RZ_SEARCH_COLLECTION_CRYPTOGRAPHIC_X509,
+	RZ_SEARCH_COLLECTION_CRYPTOGRAPHIC_ALL, ///< All algos from above.
+
 	// Always the last element to define enum size
 	RZ_SEARCH_COLLECTION_CRYPTOGRAPHIC_ENUM_SIZE,
 } RzSearchCollectionCryptographicType;
 
-RZ_API RZ_OWN RzSearchCollection *rz_search_collection_cryptographic(bool add_all_methods);
+RZ_API RZ_OWN RzSearchCollection *rz_search_collection_cryptographic();
```

Comment:
Revert this?

Reply:
I'd like to keep it this way. We have `rz_search_collection_cryptographic_add` for adding methods. Adding search method in the initialization function would delude the consistency of the design.
Meaning: After initialization of the any collection, it is expected to be empty. Search methods have to be added in a separated step.

Reply:
https://github.com/rizinorg/rizin/pull/5002